src/pages/api/fake/dataset/datasets.json

```diff
@@ -0,0 +1,21 @@
+[
+  {
+    "id": "ef347285-e52a-430d-9679-dcb76b962ce7",
+    "mainPicture": "https://images.unsplash.com/photo-1537721664796-76f77222a5d0",
+    "name": "Rhizome",
+    "type": "dataset",
+    "access": "PUBLIC",
+    "categories": ["agriculture", "open-data", "demonstrator", "dataviz", "agreste"],
+    "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed iaculis efficitur nisi in malesuada. Duis quis lectus blandit, congue urna non, rhoncus ipsum. In bibendum vel nisi nec blandit. Phasellus dictum nunc est, lobortis pellentesque libero rhoncus id. Etiam et est eu magna venenatis molestie. Nulla sapien elit, rutrum quis ligula ut, lobortis egestas lectus. Nam non sapien vitae mi viverra sagittis. Sed ornare, ligula in fermentum pulvinar, tellus ex ullamcorper est, nec pulvinar mauris nibh varius elit. In eleifend rhoncus ligula sed auctor. Nunc nec lacus et leo feugiat ultrices. Etiam dictum laoreet neque, quis mollis ex dictum a. Maecenas diam elit, placerat ac lacinia sed, ornare eget nisi. Nulla convallis ante sed ligula facilisis, vitae varius nisi fringilla. Morbi eget varius felis. Phasellus consectetur dui vel turpis commodo, eu mollis velit auctor. Etiam sed metus et nunc feugiat tristique et eu mi. Phasellus arcu nisi, ultrices quis sapien sed, ullamcorper ultricies sapien. Sed ultricies eros dui, vel blandit odio fermentum consequat. Morbi tincidunt dolor at enim dignissim, sit amet placerat metus fermentum. Pellentesque nec pharetra magna, id bibendum odio. In ante orci, sollicitudin elementum tempus a, rutrum nec felis. Cras varius ante auctor massa auctor, in consectetur ex lacinia. Phasellus bibendum turpis magna. Etiam placerat vulputate dui vel interdum. Mauris quis felis sit amet arcu posuere gravida. Fusce vehicula turpis quam, in congue nulla imperdiet quis. Quisque egestas eu odio vel imperdiet. Sed ac nunc eget nulla dignissim dignissim. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nunc mi quam, sagittis aliquam nulla a, aliquam dapibus lacus. Nunc tortor ante, porttitor vel metus eu, sodales tincidunt massa. Praesent bibendum lectus porttitor mi vestibulum, vel molestie nibh hendrerit. Integer vel mollis arcu, vitae luctus velit. Nulla vel nibh a enim gravida pharetra a in risus. Fusce commodo dui sit amet odio vehicula tristique. Aliquam erat volutpat. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas.",
```

Comment:
We need to see with @MdechampG the real content we want to be part of the dataset